Content.Server/Singularity/EntitySystems/RadiationCollectorSystem.cs

```diff
@@ -15,19 +17,37 @@ public sealed class RadiationCollectorSystem : EntitySystem
         [Dependency] private readonly IGameTiming _gameTiming = default!;
         [Dependency] private readonly PopupSystem _popupSystem = default!;
         [Dependency] private readonly SharedAppearanceSystem _appearance = default!;
+        [Dependency] private readonly SharedContainerSystem _containerSystem = default!;
+        [Dependency] private readonly IEntityManager _entityManager = default!;
 
         public override void Initialize()
         {
             base.Initialize();
             SubscribeLocalEvent<RadiationCollectorComponent, InteractHandEvent>(OnInteractHand);
             SubscribeLocalEvent<RadiationCollectorComponent, OnIrradiatedEvent>(OnRadiation);
+            SubscribeLocalEvent<RadiationCollectorComponent, ExaminedEvent>(OnExamined);
+            SubscribeLocalEvent<RadiationCollectorComponent, GasAnalyzerScanEvent>(OnAnalyzed);
+        }
+
+        private bool TryGetLoadedGasTank(EntityUid uid, out GasTankComponent? gasTankComponent)
```

Comment:
Mark the out comp with a NotNullWhen(true) attribute.

Reply:
Ooh, this tip is nice, thanks for that!